src/Genre.elm

```diff
@@ -1,9 +1,12 @@
-module Genre exposing (..)
+module Genre exposing (fromQueryString, multiSelectModel, selectionView, Genre, fromFlatList)
```

Comment:
`exposing (..)` is usually not recommended. I did it a lot when I first started because it's easy, but it's generally considered a better practice to make the exports explicit.